subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -141,6 +141,28 @@ int ull_peripheral_iso_reset(void)
 	return 0;
 }
 
+/* Use this function to de-allocate CIS/CIG resources on an aborted CIS setup
+ * ie if CIS setup is 'cancelled' after call to ull_peripheral_iso_acquire()
+ * because of a rejection of the CIS request
+ */
+void ull_peripheral_iso_dealloc(uint16_t cis_handle)
```

Comment:
```suggestion
/* Use this function to release CIS/CIG resources on an aborted CIS setup
 * ie if CIS setup is 'cancelled' after call to ull_peripheral_iso_acquire()
 * because of a rejection of the CIS request
 */
void ull_peripheral_iso_release(uint16_t cis_handle)
```

Reply:
Ack